app/uk/gov/hmrc/gform/addresslookup/AddressLookupController.scala

```diff
@@ -704,7 +707,9 @@ class AddressLookupController(
       editable = true,
       submissible = true,
       derived = false,
-      errorMessage = None
+      errorMessage = None,
```

Comment:
should we add errorMessage from the parent component as well?

Reply:
There is no AC about `errorMessage`. I'm not sure we need to add it

Reply:
@alexhurst don't you think it makes sense to mention in the ticket that if errorMessage is configured - the "manual address" page should respect that.
Also as a side note I guess the "manual address" page is not part of the error reporting. We should probably have a ticket to fix that.

Reply:
@alexhurst Actually I think there is a bigger problem here. 
```      "fields": [
        {
          "type": "postcodeLookup",
          "id": "postcode1",
          "label": "Address",
          "shortName": "Business address",
          "chooseAddressLabel": "Choose your business address",
          "confirmAddressLabel": "Confirm your business address"
        }
```
The error properties here will be applies to `postcode` page. The ticket applies them to "manual address" lookup page which is not in the form but synthetically generated and there is no way to to specify them explicitly.

Reply:
@lxol I think that is the problem with defining an errorMessage. So for this ticket defining the errorShortNames and only using them on the manual address entry page should work. I would not worry about errorMessage for now and then we can do another ticket if it is required. Or we introduce an errorMessage but that only applied to the manual address entry page.